sdk/core/azure-core/src/base64.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/core/base64.hpp"
+
+#include <azure/core/platform.hpp>
```

Comment:
Why <> here and " " above?

Reply:
It should be "" because it's in the same package.

Reply:
Yep, will fix.